Remove contradictory phrase

rootulp opened the pull request.

rootulp committed:
Remove contradictory phrase

PharkMillups reviewed (approved):
Good catch. Merging. Thanks!

PharkMillups commented:
Hmm. Seems to be a conflict somewhere. @rootulp can you fix that up?

rootulp commented:
Yep! Sorry for my delay. It looks like the file I originally updated was moved and the file extension changed.
Anyway, I fixed the conflict and updated this PR so it's ready for another review. @PharkMillups @King-of-Spades
